src/main/java/seedu/address/storage/JsonSerializableEventList.java

```diff
@@ -16,7 +16,7 @@
 @JsonRootName(value = "eventlist")
 public class JsonSerializableEventList {
 
-    public static final String MESSAGE_DUPLICATE_MODULE = "Module list contains duplicate moduke(s).";
+    public static final String MESSAGE_DUPLICATE_EVENT = "Module list contains duplicate moduke(s).";
```

Comment:
String message is wrong, I think should be "Event list contains duplicate event(s)."

Reply:
Will change it in the next push.